libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -458,18 +458,18 @@ void AP_Airspeed::setHIL(float airspeed, float diff_pressure, float temperature)
     state[0].healthy = true;
 }
 
-bool AP_Airspeed::use(uint8_t i) const
+int AP_Airspeed::use(uint8_t i) const
```

Comment:
if this now takes on more than true/false then it should return an enum

Reply:
Returned it back to returning a boolean in the rejigged code.